pavelib/utils/test/suites/nose_suite.py

```diff
@@ -120,7 +120,7 @@ def __enter__(self):
     @property
     def cmd(self):
         cmd = (
-            './manage.py {system} test --verbosity={verbosity} '
+            './manage.py {system} --contracts test --verbosity={verbosity} '
```

Comment:
If we're excluding it in manage.py, why are we adding it here? Sorry if this is obvious to you...not obvious to me though.